lightning/src/util/events.rs

```diff
@@ -350,8 +350,8 @@ pub enum Event {
 		/// [`ChannelManager::create_channel`]: crate::ln::channelmanager::ChannelManager::create_channel
 		user_channel_id: u128,
 	},
-	/// Indicates we've received (an offer of) money! Just gotta dig out that payment preimage and
-	/// feed it to [`ChannelManager::claim_funds`] to get it....
+	/// Indicates that a payment is inbound and needs to be claimed via calling
```

Comment:
Maybe this makes it clear:
```suggestion
	/// Indicates that we've been offered a payment and it needs to be claimed via calling
```

Reply:
Went with that and squashed again.